templates/client-auth-n.cfg.xml.erb

```diff
@@ -14,13 +14,21 @@
   <%- end -%>
 <%- end -%>
 <%- if @ignore_tenant_roles -%>
-    <ignore-tenant-roles>
+        <ignore-tenant-roles>
   <% @ignore_tenant_roles.each do |ignore_tenant_role| -%>
-        <role><%= ignore_tenant_role %></role>
+            <role><%= ignore_tenant_role %></role>
   <%- end -%>
-    </ignore-tenant-roles>
+        </ignore-tenant-roles>
 <%- end -%>
     </openstack-auth>
+
+<%- if @token_expire_feed -%>
+    <atom-feeds check-interval="<%= @token_expire_feed['interval'] -%>">
+        <rs-identity-feed id="identity-token-revocation-feed" uri="<%= @token_expire_feed['feed_url'] -%>" 
+                          isAuthed="true" user="<%= @auth['user'] -%>" password="<%= @auth['pass'] -%>"/>
```

Comment:
Not that it affects me, but would it make sense to use a user/password passed as part of the token_expire_feed? Thus allowing for a different user (less permissions) for this functionality.